### PR TITLE
Rollout Explore Case Data [Beta]

### PR DESCRIPTION
**Pull request by biyeun**
- Creates a new feature preview for ECD
- adds a management command to bulk rollout the feature preview for eligible domains
- updates copy
- adds an announcement bubble for newly enabled ECD feature previews

### CODE REVIEW

File: corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html [hunk 1]
```diff
@@ -72,7 +72,8 @@
   </p>
   <p class="help-block">
     {% blocktrans %}
-      This feature allows you to quickly discover the cases of your interest.<br />
+      This feature allows you to quickly explore your case data for ad-hoc data
+      queries or to identify unclean data.<br />
       This is a Preview Feature.
```
Comment (proteusvacuum) [inline corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html]: nit: elsewhere we call these "Feature Preview"s

File: corehq/feature_previews.py [hunk 2]
```diff
@@ -133,3 +158,39 @@ def enable_callcenter(domain_name, checked):
...
+    is_enterprise_eligible = ECD_PREVIEW_ENTERPRISE_DOMAINS.enabled_for_request(request)
+    is_pro_or_advanced = request.plan.plan.edition in [
+        SoftwarePlanEdition.ADVANCED,
+        SoftwarePlanEdition.PRO
+    ]
+
+    return is_migrated and (is_pro_or_advanced or is_enterprise_eligible)
+
+
+def save_ecd_preview(domain_name, _checked):
```
Comment (proteusvacuum) [inline corehq/feature_previews.py]: nit: this isn't very descriptive of what this function actually does - maybe something like `clear_dropdown_cache`?

**Review by proteusvacuum — APPROVED** [review]
:tada:

File: corehq/feature_previews.py (hunk 2 above)
Comment (biyeun) [reply corehq/feature_previews.py]: updated. thanks!

File: corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html (hunk 1 above)
Comment (biyeun) [reply corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html]: haha. yep. updated--thanks!

whoops

**Pull request merged**